com.unity.cinemachine/Editor/Samples/ExposeHDRPInternals/MaterialUpgradeHelper.cs

```diff
@@ -0,0 +1,26 @@
+#if HDRP_1_OR_NEWER
+using System.Collections.Generic;
+using UnityEditor.Rendering;
+using UnityEditor.Rendering.HighDefinition;
+using UnityEngine;
+
+namespace Unity.Cinemachine.Editor
```

Comment:
Is this the right namespace to use?  I'm thinking that if we're extending HDRP, then maybe it should be in the UnityEditor.Rendering.HighDefinition namespace.

Reply:
Could be. I was not sure what namespace to use here, because yes, we are extending HDRP. 

It could also be, namespace `ExposedHDRP`.

Reply:
`UnityEditor.Rendering.HighDefinition` makes the most sense. Changed it to that.